renovate.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:recommended", ":dependencyDashboardApproval"],
+  "vulnerabilityAlerts": {
+    "assignees": ["kirchsuSICKAG", "klaarseSICKAG", "schilchSICKAG"],
+    "autoApprove": true,
+    "automerge": true,
+    "automergeStrategy": "squash",
+    "automergeType": "pr"
+  },
+  "draftPR": true
```

Comment:
Does `draftPR` work together with `automerge`?

And I think it's important to set https://docs.renovatebot.com/configuration-options/#rebasewhen that Renovate doesn't rebase regularly, especially when it's not a draft PR anymore?

Reply:
The automerge is only on the vulnerabilities. I wanted the draft PRs on the normal dependency updated (if creating a PR over the issue dashboard). But actually till now I don´t know if it works correctly, because draft PR can not be created on private repos, where I tested the config.

It seems like with this config Renovate does not rebase other PRs. I tested it on the other repo and the open dependency update PR was not rebased. I think this is because we don`t have the auto generation of an PR but only on dashboard approval. But to be sure I could just add it.